### DISCUSSION TITLE
How to disable ES?

### DISCUSSION THREAD

**Discussion by flavioschuindt** [Netflix/conductor#2472] [Q&A]
Hi, guys.

So, how can I disable ES at all? I know that you can create a dummy implementation of it that does nothing and make conductor use it, but then you will lose all the search capabilities. ES is a huge footprint and having it really impacts the deployment. Is there a way we can totally remove it without breaking the search capability?

P.S.: This is for conductor version 2.X.

Thank you!

**Comment by kishorebanala** [comment]
@flavioschuindt I'm afraid you can't disable ES and have search features. Alternatively, you can implement search interface to query against your primary db, and then disable ES.
